static/bidder-info/vrtcal.yaml

```diff
@@ -7,13 +7,14 @@ capabilities:
     mediaTypes:
       - banner
       - video
+      - native
   site:
     mediaTypes:
       - banner
       - video
+      - native
```

Comment:
Requesting you to please add the json test case for this change.

Reply:
Native test cases added